src/tcpreplay_opts.def

```diff
@@ -663,6 +663,15 @@ EOVersion;
     doc         = "";
 };
 
+flag = {
+    name        = batch-size;
+    arg-type    = number;
+    arg-range   = "1->4096";
```

Comment:
this range is enforced by the application, there is no need for additional check